indico/types/base.py

```diff
@@ -28,18 +37,22 @@ def valid_type(v):
     )
 
 
+# TODO: all the introspection here class makes the typing sus
```

Comment:
I wonder if we can do away with this class at some point

Reply:
moving to pydantic is my next pet project

Reply:
cooldown collab?